conf.py

```diff
@@ -75,3 +75,50 @@
 }
 
 panels_add_bootstrap_css = False
+
+# -- Custom scripts -----------------------------------------
+import pandas as pd
+from pathlib import Path
+
+# Pull list of hub repositories
+hub_table = "https://docs.google.com/spreadsheets/d/e/2PACX-1vTpwlQL0Moz-ej-ADNfc6JTxfTje5GBQD1o0DE66atYi2ulWan3leEiCC20epzTyHVWIdomxp9Jt70D/pub?gid=0&single=true&output=csv"
+df = pd.read_csv(hub_table)
+Path("tmp").mkdir(exist_ok=True)
+path_table = Path("tmp/repo-deploy-table.csv")
+if not path_table.exists():
+    df.to_csv(path_table, index=None)
+
+# Pull latest list of communities served by pilot-hubs/
+import requests
+from yaml import safe_load
+
+yaml_list = [
+    "https://raw.githubusercontent.com/2i2c-org/pilot-hubs/master/config/hubs/2i2c.cluster.yaml",
+    "https://raw.githubusercontent.com/2i2c-org/pilot-hubs/master/config/hubs/cloudbank.cluster.yaml",
+    "https://raw.githubusercontent.com/2i2c-org/pilot-hubs/master/config/hubs/paleohack.cluster.yaml",
+]
```

Comment:
Don't you want to fetch all the `.cluster.yaml` here? Otherwise, this one will get outdated when we have a new cluster.

Reply:
Are there others that I'm missing?

Reply:
> Ah yeah - maybe a better approach would be to use the github API to grab these files, rather than hard code the yaml files here.

Yep...

> Are there others that I'm missing?

I don't think so, just thinking into the future...